src/main/java/com/dev/cinema/dao/GenericDao.java

```diff
@@ -3,11 +3,15 @@
 import com.dev.cinema.exceptions.DataProcessingException;
 import com.dev.cinema.util.HibernateUtil;
 import java.util.List;
+import org.apache.log4j.Logger;
 import org.hibernate.Session;
 import org.hibernate.Transaction;
 import org.hibernate.query.Query;
 
 public interface GenericDao<T> {
+
+    Logger logger = Logger.getLogger(GenericDao.class);
```

Comment:
access modifiers

Reply:
it`s interface, so it  not supposed to, isn`t it?

Reply:
you are right!